Documentation/zenko 195 1.2 remerge 2nd try

wabernat opened the pull request:
I am hoping that this PR introduces the documentation for S3C OOB updates that were sidelined in 1.1.0

This fixes #ZENKO-195 and ZENKO-1801

bert-e commented:
# Hello wabernatscality,



My role is to assist you with the merge of this
pull request. Please type `@bert-e help` to get information
on this process, or consult the [user documentation](
https://eve.devsca.com/github/scality/zenko/bert-e/doc/user).





*Status report is not available.*

bert-e commented:
# Incorrect fix version



The `Fix Version/s` in issue ZENKO-195 contains:


* `1.2.0`


Considering where you are trying to merge, I expected to find:


* `1.1.0`

* `1.2.0`


Please check the `Fix Version/s` of ZENKO-195, or the target
branch of this pull request.

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* 2 peers





Peer approvals *must* include at least 1 approval from the following list:

* @nicolast

* @tmacro

* @ssalaues

* @vrancurel

wabernat committed:
documentation: ZENKO-195_1.2_remerge_2nd_try

This reintroduces the bucket ingestion metrics content removed
in ZENKO-1801.

It's a hot Git mess, and I'm not sure how I got here.

documentation: ZENKO-1765_restructure_zenko_cli_docs_in_ops-ref

This is not a one-liner.

This commit reworks the Zenko CLI chapter, introduces an improved
Backbeat CLI experience, fixes a lot of small problems with the
language and appearance of the document.

Several subsections were moveed from Reference to Operations and
vice-versa to rationalize the documentation.

documentation/ZENKO-195_1.2_remerge_2nd_try

cherry-picked 4397ce8

added index.rst

fixed bad internal link.

fixed a bad spacing.

ghost commented:
There were the following issues with this Pull Request

* Commit: bc01cd0f061ed1327e8eae8fdcb17837554ef150
  - ✖ message may not be empty
  - ✖ type may not be empty


You may need to [change the commit messages][ref] to comply with the repository contributing guidelines.

--------

🤖 This comment was generated by [**commitlint[bot]**][repo]. Please report issues [here][issues].

Happy coding!

[ref]: https://help.github.com/articles/changing-a-commit-message/
[repo]: https://github.com/z0al/commitlint-bot
[issues]: https://github.com/z0al/commitlint-bot/issues

wabernat commented:
@bert-e approve

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* 2 peers





Peer approvals *must* include at least 1 approval from the following list:

* @nicolast

* @tmacro

* @ssalaues

* @vrancurel










*The following options are set:* **approve**

dora-korpar commented on docs/docsource/reference/backbeat/crr_retry.rst:
```diff
@@ -15,7 +15,7 @@ This GET request retrieves a listing of failed operations at a site. Use
 and to retrieve the entire listing.
 
-**Request:** GET /_/backbeat/api/_/crr/failed?sitename=<site>&marker=<next-marker>
+**Request:** GET/_/backbeat/api/crr/failed?sitename=<site>&marker=<next-marker>
```
Should there be a space after the method `GET`? The other requests have one

wabernat replied:
Good spot.

bert-e commented:
# In the queue



The changeset has received all authorizations and has been added to the
relevant queue(s). The queue(s) will be merged in the target development
branch(es) as soon as builds have passed.

The changeset will be merged in:


* :heavy_check_mark: `development/1.2`



The following branches will **NOT** be impacted:

* `development/0.9`
* `development/1.0`
* `development/1.1`



There is no action required on your side. You will be notified here once
the changeset has been merged. In the unlikely event that the changeset
fails permanently on the queue, a member of the admin team will
contact you to help resolve the matter.

*IMPORTANT*

Please do not attempt to modify this pull request.

* Any commit you add on the source branch will trigger a new cycle after the
  current queue is merged.
* Any commit you add on one of the integration branches will be **lost**.

If you need this pull request to be removed from the queue, please contact a
member of the admin team now.




*The following options are set:* **approve**

bert-e commented:
# 


I have successfully merged the changeset of this pull request
into targetted development branches:


* :heavy_check_mark: `development/1.2`



The following branches have **NOT** changed:

* `development/0.9`
* `development/1.0`
* `development/1.1`



Please check the status of the associated issue ZENKO-195.

Goodbye wabernatscality.
